src/MeetingsSDKAdapter/controls/ShareControl.js

```diff
@@ -48,8 +48,20 @@ export default class ShareControl extends MeetingControl {
       tooltip: 'Stop sharing content',
     };
 
+    const disabled = {
+      ID: this.ID,
+      type: 'TOGGLE',
+      state: MeetingControlState.DISABLED,
+      icon: 'share-screen-presence-stroke',
+      text: 'Start sharing',
+    };
+
```

Comment:
remove this blank line, as the states inactive and active are not separated by a blank line - or separate those with a blank line, too